Automattic/wp-calypso
# Use FSE flags on recommended themes filter

#### Changes proposed in this Pull Request

* Use `wpcom/v2/sites/%s/block-editor` endpoint to determine if the site is eligible for FSE settings. With that setting, we can choose if we want to recommend block-based themes or regular ones.

#### Testing instructions

- Run this branch within a beta-enrolled site (FSE);
- Open the themes page and check that your recommend themes are block-based ones.



Related to #56373.
